apps/omg_performance/lib/performance.ex

```diff
@@ -211,6 +264,28 @@ defmodule OMG.Performance do
     :ok
   end
 
+  @spec setup_standard_exit_perftest(map()) :: map()
+  defp setup_standard_exit_perftest(opts) do
+    exit_for = Map.get(opts, :exit_for)
+
+    utxos =
+      case exit_for do
+        %{addr: addr} ->
+          addr
+          |> ByzantineEvents.get_exitable_utxos()
+          |> Enum.map(& &1.utxo_pos)
+          |> Enum.shuffle()
+          |> Enum.take(opts.exits_per_user)
+
+        nil ->
+          ByzantineEvents.Generators.stream_utxo_positions()
+          |> Enum.take(opts.exits_per_user)
```

Comment:
single pipe

Reply:
you cannot leave it unchanged because we agreed on specific rules we will use for the codebase.
I suggest you do:
```suggestion
          stream_utxo_positions = ByzantineEvents.Generators.stream_utxo_positions()
          Enum.take(stream_utxo_positions, opts.exits_per_user)
```